ticketutil/servicenow.py

```diff
@@ -41,11 +53,19 @@ def __init__(self, url, project, auth=None, ticket_id=None):
 
```

Comment:
Line 40 above should say ticket_id and ticket number instead of ticked_id and ticked number